sbi/inference/posteriors/direct_posterior.py

```diff
@@ -583,14 +599,18 @@ def posterior_potential(
             self.device
         )
 
+        theta, log_abs_det = self.transform(theta)
```

Comment:
I would make it more explicit and call it `transformed_theta` or `tf_theta`

Reply:
to be clear, the incoming theta is the transformed one, no? so it should be 
`theta, ladj = self.transform(tf_theta)`?

Reply:
yes, that's right

Reply:
It's a bit tricky. The code is
```
def posterior_potential(self, theta: Union[Tensor, np.array], track_gradients: bool = False):
        theta = ensure_theta_batched(torch.as_tensor(theta, dtype=torch.float32)).to(
            self.device
        )
        theta, log_abs_det = self.transform(theta)
```
So, we would have to rename the incoming `theta` to `theta_tf`. Not sure if we should really do that. For now I'll just add comments.